src/problem1.js

```diff
@@ -1,6 +1,94 @@
 function problem1(pobi, crong) {
-  var answer;
-  return answer;
+  let maxPobiNumber = 0;
+  let maxCrongNumber = 0;
+  const isPobiWin = () => maxPobiNumber > maxCrongNumber;
+
+  if (!isValidArray(pobi) || !isValidArray(crong)) return -1
+
+  maxPobiNumber = getMaxFromTwoNumber(getMaxFromAddSum(pobi),getMaxFromMultiplySum(pobi));
+  maxCrongNumber = getMaxFromTwoNumber(getMaxFromAddSum(crong),getMaxFromMultiplySum(crong));
+
+  if (isResultDraw(maxPobiNumber, maxCrongNumber)) return 0; 
+  return isPobiWin() ? 1 : 2; 
+}
+
+/**
+* 길이가 2인 배열을  넣으면 그 합을 리턴해주는 함수.
+* @param {array} arr 입력된 배열
+* @returns {number} 덧셈 결과 리턴
+*/
+const getMaxFromAddSum = arr => {
```

Comment:
JavaScript에서도 type을 **기본값을 통해 명시**할 수 있습니다!
아래와 같이 수정한다면 `arr.`까지 입력했을 때 Array 관련 메소드들이 자동완성 된답니다 :)
```suggestion
const getMaxFromAddSum = (arr = []) => {
```

Reply:
우와.. 몰랐는데 정말 좋은 팁입니다..!
꿀팁 공유 감사합니다!

Reply:
하지만 다른 분의 의견을 들었을 때 기본값을 명시하면 해당 값이 optional해져서 `getMaxFromAddSum()`으로도 접근이 가능해집니다!! 
빈 배열을 처리할 코드를 추가적으로 작성하거나 optional한 함수라면 기본값을 명시해도 괜찮지만, 무조건 파라미터가 필요한 함수라면 기본값을 명시하는 것은 오히려 독이 되지 않을까!! 생각이 듭니다 ^_ㅠ

Reply:
이 꿀팁으로 기본값을 명시하면 optional해지기에,
파라미터가 필수라면 고려해보아라.

잊지않고 찾아와주셔, A/S까지 확실하게 해주셔서 감사합니다 ㅠㅠ👍